test/rruleset.test.ts

```diff
@@ -727,4 +727,45 @@ describe('RRuleSet', function () {
     expect(() => set.rdate('foo' as any)).to.throw()
     expect(() => set.exdate('foo' as any)).to.throw()
   })
-})
+
+  describe('getters', () => {
+    it('rrules()', () => {
+      let set = new RRuleSet();
+      let rrule = new RRule({
+        freq: RRule.YEARLY,
+        count: 2,
+        dtstart: parse('19600101T090000'),
+        tzid: 'America/New_York'
+      });
+      set.rrule(rrule);
+
+      expect(set.rrules().map(e => e.toString())).eql([rrule.toString()]);
+    });
```

Comment:
one small style nitpick: can we make sure there are blank lines in between `it` blocks? thanks!

Reply:
Done